lib/engines/content_block_manager/app/views/admin/errors/bad_request.html.erb

```diff
@@ -5,6 +5,12 @@
 
 <div class="govuk-grid-row govuk-!-margin-bottom-0">
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">Please try again by selecting the browser&rsquo;s back button or <%= link_to "raise a support request", ContentBlockManager.support_url, class: "govuk-link govuk-link--no-visited-state" %></p>
+    <p class="govuk-body">Please try again by selecting the browser’s back button or email us at
+      <%= mail_to(
+            ContentBlockManager.support_email_address,
+            ContentBlockManager.support_email_address,
```

Comment:
If the link text is the same as the email address I think you can just specify it once